Closing notes on project feast (#91)
comit-network/comit.network: blog/2020-11-06-project-feast-closing-notes.md

---
title: "Fail⬅️⬅️⬅️⬅️Learning faster"
author: franck
author_url: https://github.com/d4nte
author_image_url: https://avatars1.githubusercontent.com/u/300805
tags: [bitcoin,dai,atomic swap,ambrosia]
---

import useBaseUrl from '@docusaurus/useBaseUrl';

# Ambrosia

We recently released [ambrosia](./2020-10-12-ambrosia-alpha-release.md), a GUI allowing BTC to DAI peer-to-peer and non-custodial trading powered by COMIT.
This release marked our change of focus from providing tools to developer to directly targeting end users. 

We spent the last weeks demonstrating this MVP to several users, we had interesting feedback and conversation with various user profiles, from trader to hodler.

<!--truncate-->

At this point of time, we felt that BTC&lt;&gt;DAI dex trading may not be what users are currently after and hence decided to drop the project.

> While the development efforts which lead to this MVP did not lead to a successful product, time was not wasted.

Reviewer: It sounds like a comma is missing: 
```suggestion
While the development efforts, which lead to this MVP, did not lead to a successful product, time was not wasted.
```

Author: I simplified the sentence.

We made major learnings that will allow us to move faster, for example, our blockchain monitoring solution was not up to scratch for a mainnet environment.
You can find more details of our learnings in the [closing notes](https://lists.comit.network/pipermail/comit-dev/2020-October/000017.html).

> # wBTC&lt;&gt;BTC

Reviewer: If you don't want to have these weird characters, you could write this: `BTC/DAI` 
```suggestion
# wBTC/BTC
```

Reviewer: Actually this looks quite good as it is.

![image](https://user-images.githubusercontent.com/224613/98485725-a41b5880-226c-11eb-8820-cbf04e48c6a3.png)

Several users have expressed interest in a wrapped Bitcoin to Bitcoin atomic swap trading platform.

> The drive behind it is two folds: avoid the option problem and create an open market for ERC20 wrapped BTC.

Reviewer: ```suggestion
The drive behind it is twofold: avoid the option problem and create an open market for ERC20 wrapped BTC.
```

Reviewer 2: I am always a bit hesitant with using our internal language within public posts. I doubt that the average user has an idea of what "the option problem" is.

However, I don't think it is worth explaining in this post so feel free to leave it in.

Author: Yes, I thought exactly the same @thomaseizinger but wasn't sure how to handle it :)

Depositing and withdrawing wrapped Bitcoin can have some restrictions depending on the provider: fees, KYC, high confirmation requirements.

Because of the demand, we forked Ambrosia to provide a wBTC&lt;&gt;BTC version.

> We chose wBTC because it is simply the most popular wrapped Bitcoin token.

Reviewer 2: Is this claim actually true?

Reviewer: Yes, if popularity measured by trading volume.
